Formula/php@8.0.rb

```diff
@@ -0,0 +1,461 @@
+class PhpAT80 < Formula
+  desc "General-purpose scripting language"
+  homepage "https://www.php.net/"
+  # Should only be updated if the new version is announced on the homepage, https://www.php.net/
+  url "https://www.php.net/distributions/php-8.0.13.tar.xz"
+  mirror "https://fossies.org/linux/www/php-8.0.13.tar.xz"
+  sha256 "cd976805ec2e9198417651027dfe16854ba2c2c388151ab9d4d268513d52ed52"
+  license "PHP-3.01"
+
+  livecheck do
+    url "https://www.php.net/downloads"
+    regex(/href=.*?php[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  keg_only :versioned_formula
+
```

Comment:
```suggestion

deprecate! date: "2023-11-26", because: :versioned_formula

```

See https://www.php.net/supported-versions.php